src/kintone_client/user.cljc

```diff
@@ -0,0 +1,212 @@
+                 (some? offset) (assoc :offset offset)
+                 (some? size) (assoc :size size))]
+    (pt/-get conn (pt/-user-api-url conn path/users) {:params params})))
+
+(defn add-users
+  "Adds users to a kintone environment.
+
+  users - See API reference regarding user format.
+          sequence of map"
+  [conn users]
+  (pt/-post conn (pt/-user-api-url conn path/users) {:params {:users users}}))
+
+(defn update-users
+  "Updates users of a kintone environment.
+
+   users - See API reference regarding user format.
+           sequence of map"
+  [conn users]
+  (pt/-put conn (pt/-user-api-url conn path/users) {:params {:users users}}))
+
+(defn delete-users
+  "Deletes users from a Kintone environment.
+
+  user-codes - An array of user codes of users to be deleted.
+               sequence of string"
+  [conn user-codes]
+  (pt/-delete conn (pt/-user-api-url conn path/users) {:params {:codes user-codes}}))
+
+(defn update-user-codes
+  "Updates user codes of users.
+
+  codes - sequence of map
+
+    [] :currentCode - The current User Code (log-in name).
+                      string
+
+    [] :newCode - The new User Code (log-in name).
+                  string"
+  [conn codes]
+  (pt/-put conn (pt/-user-api-url conn path/user-codes) {:params {:codes codes}}))
+
+(defn get-organizations
+  "Gets information of organizations.
+
+  opts
+
+    :ids - A list of Organization IDs.
+           sequence of integer
+
+    :codes - A list of Organization Codes.
+             sequence of string
+
+    :offset - The offset.
+              integer
+
+    :size - The maximum number of Organization information to get.
+            integer"
+  [conn {:keys [ids codes offset size]}]
+  (let [params (cond-> {}
+                 (some? ids) (assoc :ids ids)
+                 (some? codes) (assoc :codes codes)
+                 (some? offset) (assoc :offset offset)
+                 (some? size) (assoc :size size))]
+    (pt/-get conn (pt/-user-api-url conn path/organizations) {:params params})))
+
+(defn add-organizations
+  "Adds organizations to a kintone environment.
+
+  organizations - See API reference regarding organization format.
+                  sequence of map"
+  [conn organizations]
+  (pt/-post conn (pt/-user-api-url conn path/organizations) {:params {:organizations organizations}}))
+
+(defn update-organizations
+  "Updates organizations of a Kintone environment.
+
+  organizations - See API reference regarding organization format.
+                  sequence of map"
+  [conn organizations]
+  (pt/-put conn (pt/-user-api-url conn path/organizations) {:params {:organizations organizations}}))
+
+(defn delete-organizations
+  "Deletes organizations from a Kintone environment.
+
+  org-codes - An array of organization codes of organization to be deleted.
+              sequence of string"
+  [conn org-codes]
+  (pt/-delete conn (pt/-user-api-url conn path/organizations) {:params {:codes org-codes}}))
+
+(defn update-org-codes
```

Comment:
I like `update-organization-codes`. I think it to be redundant but the priority of the name unity is more important.